first-party/jib-native-image-extension-gradle/README.md

```diff
@@ -0,0 +1,42 @@
+# Jib GraalVM Native Image Extension
+
+This extension containerizes a [GraalVM native-image](https://www.graalvm.org/docs/reference-manual/native-image/) application configured with [Native Image Gradle Plugin](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html).
+
+The extension expects the `org.graalvm.buildtools.native` to do the heavy lifting of generating a "native image" (with the `nativeCompile` task). (The "image" in "native image" refers to an executable binary, not a container image.) Then the extension simply copies the binary, say, `<project root>/build/native/nativeCompile/com.example.mymainclass`, into a container image and sets executable bits. As of the moment, unlike its maven counterpart, it does `not` auto-set the container image entrypoint to the binary, say, `/app/com.example.mymainclass` so you will have to manually configure `<container><entrypoint>` in the main Jib configuration.
```

Comment:
Ah I may be mistable but it looks like we do auto-set the entrypoint to the executable in line 120 of `JibNativeImageExtension`:
```
if (jibContainer.getEntrypoint() == null
        || Objects.requireNonNull(jibContainer.getEntrypoint()).isEmpty()) {
      planBuilder.setEntrypoint(Collections.singletonList(targetExecutable.toString()));
    }
```

Reply:
Ah got it. In that case how about we remove that line?:
```suggestion
The extension expects the `org.graalvm.buildtools.native` to do the heavy lifting of generating a "native image" (with the `nativeCompile` task). (The "image" in "native image" refers to an executable binary, not a container image.) Then the extension simply copies the binary, say, `<project root>/build/native/nativeCompile/com.example.mymainclass`, into a container image and sets executable bits. 
```

Reply:
Put this back to the way it is in the maven version. (Rather than remove it all)